app.rb

```diff
@@ -104,7 +103,6 @@ def create_a_teacher
     teacher = Teacher.new(age: age, name: name, specialization: specialization)
     @people.push(teacher)
     puts 'Teacher created successfully'
-    puts
     choose_a_number
   end
```

Comment:
- [ ] in the `create_a_teacher` method you are printing to the console getting information from the user and saving the information this is not in line with the single responsibility rule, have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more information.